cfme/automate/import_export.py

```diff
@@ -209,7 +210,8 @@ def import_domain_from(self, domain_from, domain_into=None, toggle_all=True):
             {"import_into": domain_into, "import_from": domain_from, "toggle_all": toggle_all}
         )
         file_view.commit.click()
-        return self.appliance.collections.domains.instantiate(name=domain_from)
+        return self.appliance.collections.domains.instantiate(name=domain_from,
```

Comment:
If you just want to add a description for the added domain you can do in test. 

 domain.update(description="some description"